src/scanoss/threadeddependencies.py

```
@@ -76,8 +92,15 @@ def responses(self) -> Dict:
                 return resp
         return None
 
-    def run(self, what_to_scan: str = None, deps_file: str = None, wait: bool = True, dep_scope: SCOPE =  None,
-            dep_scope_include: str = None, dep_scope_exclude: str = None) -> bool:
+    def run(
```

Ruff (0.8.2)

Too many arguments in function definition (6 > 5)

(PLR0913)